css/index.css

```diff
@@ -143,4 +266,79 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+a:hover {
+    color:blue;
+}
+
+/* CSS for product specific page */
+
+.middle-container {
+	display:flex;
+	flex-wrap:wrap;
+	justify-content: space-between;
+    padding-top:30px;
```

Comment:
Look into plugins like `Beautify` or `Prettier` they'll automatically format the CSS spacing for you.